### ISSUE TITLE
It's a bummer that you need to install GhostScript in OS

### ISSUE THREAD

**Issue by hovek** [dlemstra/Magick.NET#1346]
### Describe the solution you'd like

This is a deal breaker for me because I don't want to force users to install additional software to be able to use mine.

**Comment by dlemstra** [comment]
You are right that it would be easier if this was included but that is not possible due to licensing issues. If you want to use Ghostscript commercially you will need to pay for a license.
